src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/PerformerNameAndAddress.java

```diff
@@ -38,24 +50,26 @@ public static PerformerNameAndAddress fromString(String edifactString) {
 
         String[] keySplit = Split.byPlus(edifactString);
         String[] colonSplit = Split.byColon(keySplit[2]);
-        String performerID = colonSplit[0];
+        String identifier = colonSplit[0];
+
+        final boolean isOrganization = StringUtils.isBlank(identifier);
+        if (isOrganization) {
+            // if identifier is blank - organization/department
+            String partyName = keySplit[4].replaceAll("\\?'", "'");
```

Comment:
We discussed this before and agreed that the segments should hold the raw data and the mappers would be responsible for converting and escaping them.